sdk/instrumentation/library.go

```diff
@@ -22,12 +22,4 @@ For more information see
 package instrumentation // import "go.opentelemetry.io/otel/sdk/instrumentation"
 
 // Library represents the instrumentation library.
-type Library struct {
-	// Name is the name of the instrumentation library. This should be the
-	// Go package name of that library.
-	Name string
-	// Version is the version of the instrumentation library.
-	Version string
-	// SchemaURL of the telemetry emitted by the library.
-	SchemaURL string
-}
+type Library Scope
```

Comment:
```suggestion
type Library = Scope
```

I'm pretty sure we want a type alias here rather than a definition.  This makes `Library` another name for the `Scope` struct, not another type.  See the second example [here](https://go.googlesource.com/exp/+/master/apidiff/README.md#exported-types).

Reply:
Ah! Thank you for the explanation! PR updated.